fs/filter.go

```diff
@@ -194,8 +197,14 @@ func NewFilter() (f *Filter, err error) {
 			if err != nil {
 				return nil, err
 			}
+			foundExcludeRule = true
 		}
 	}
+
+	if addImplicitExclude && foundExcludeRule {
+		log.Printf("Using --filter is recommended instead of both --include and --exclude as the order they are parsed in is indeterminate")
```

Comment:
I think that should be a `Noticef(nil,...` rather than a `log.Printf`

Reply:
Done

Reply:
OK, I will write something in `docs/content/filtering.md`